advcpmv-0.8-8.32.patch

```diff
@@ -13,7 +14,57 @@ diff -aur coreutils-8.32/src/copy.c coreutils-8.32-patched/src/copy.c
 +  struct stat src_open_sb;
 +};
 +
-+/* Begin progress Mod*/
++char * escape_double_quotes ( char * _cUnescapedString )
++{
++  char * _cEscapedString;
++  _cEscapedString = (char *) calloc( sizeof(char) * strlen(_cUnescapedString) * 2, sizeof(char) );
```

Comment:
Also you're not keeping track of how much you have allocated inside the loop so it's possible to overrun the buffer.

Reply:
This is a problem in the special case only, that all characters would need to be escaped. But you are right, it's better to reserve another char space. I'll fix it.

Reply:
By switching to your spawn function, this is not necessary anymore, as the escape function is now obsolete and removed.